images-list

```diff
@@ -3,12 +3,14 @@
 # https://github.com/rancher/image-mirror/pull/27
 #
 # LIST FORMAT: <SOURCE> <DESTINATION> <TAG>
+banzaicloud/fluentd rancher/banzaicloud-fluentd v1.10.4-alpine-2-debug
```

Comment:
we should mirror the debug image the same as the actual image `v1.11.2-alpine-2-debug`

Reply:
Good catch. Fixed and I'll note this in the issue filed.

Reply:
This doesn't exist now: https://hub.docker.com/r/banzaicloud/fluentd/tags?page=1&name=v1.11.2-alpine-2

Reply:
Interesting... I'll touch base with Jacob offline. Doesn't look like there are any ```fluentd``` debug images at the moment.

Reply:
Confirmed with @paynejacob. The ```debug``` images are [only offered for fluentbit](https://banzaicloud.com/docs/one-eye/logging-operator/troubleshooting/#debug-version-of-the-fluentbit-container), not fluentd. Adjusted the PR accordingly.